.travis.yml

```diff
@@ -46,6 +45,7 @@ install: MAVEN_OPTS='-Xmx3000m' travis_wait 15 ${MVN} clean install -q -ff ${MAV
 stages:
   - name: test  # jobs that do not specify a stage get this default value
     if: type != cron
+  - name: it
```

Comment:
nit: should we call this 'integration-tests' or something so it looks a bit more descriptive on web stuffs?
<img width="1231" alt="Screen Shot 2021-01-25 at 2 20 45 PM" src="https://user-images.githubusercontent.com/1577461/105773297-8929bd80-5f18-11eb-84b4-8821a66f4b5f.png">

although I do kind of like it like this too, so I'm +1 on this PR either way

Reply:
I like your suggestions. Trying to find the right balance and the right name for the stages is tough.

I could move all the jdk 11 tests into the second stage, but then I can't think of what the name should be. I thought about slow / fast. But right now `intelliJ inspections` and the `indexing modules tests` take ~ 40 minutes, which is actually longer than most integration tests 😅 

I think this split is good enough as a starting point, but would be extra happy if someone else proposed a more optimal split.